contracts/DappCampNFT.sol

```diff
@@ -6,30 +6,79 @@ import "@openzeppelin/contracts/utils/Strings.sol";
 import "@openzeppelin/contracts/utils/Base64.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
-
 contract DappCampNFT is ERC721Enumerable, Ownable {
     uint256 public MAX_MINTABLE_TOKENS = 5;
 
     constructor() ERC721("DappCamp NFT", "DCAMP") Ownable() {}
 
     string[] private collection = [
-        ""
+        "action",
+        "faithful",
+        "navy",
+        "gold",
+        "maroon",
+        "pick 6",
+        "ball",
+        "OKC",
+        "believe"
     ];
 
     function random(string memory input) internal pure returns (uint256) {
-        
+        // take the letter of the alphabet and the last letter of the alphabet and divides by the length of the string
+        return uint256(keccak256(abi.encodePacked(input)));
     }
 
-    function pluck(uint256 tokenId, string memory keyPrefix, string[] memory sourceArray) internal pure returns (string memory) {
-
+    function pluck(
+        uint256 tokenId,
+        string memory keyPrefix,
+        string[] memory sourceArray
+    ) internal pure returns (string memory) {
+        uint256 rand = random(string(abi.encodePacked(keyPrefix, toString(tokenId))));
+        string memory output = sourceArray[rand % sourceArray.length];
+        return output;
     }
 
-    function tokenURI(uint256 tokenId) override public view returns (string memory) {
+    function tokenURI(uint256 tokenId)
+        public
+        view
+        override
+        returns (string memory)
+    {
+        string memory output = string(abi.encodePacked(pluck(tokenId, "Della", collection)));
+        string memory json = Base64.encode(bytes(string(abi.encodePacked('{"name": "Della NFT #', toString(tokenId), '", "description": "This is an NFT that Della made. Because she got it like that.", "image": "data:image/svg+xml;base64,', Base64.encode(bytes(output)), '"}'))));
+        output = string(abi.encodePacked('data:application/json;base64,', json));
 
+        return output;
     }
 
     function claim(uint256 tokenId) public {
-        require(tokenId > 0 && tokenId < MAX_MINTABLE_TOKENS, "Token ID invalid");
+        require(
+            tokenId > 0 && tokenId < MAX_MINTABLE_TOKENS,
+            "Token ID invalid"
+        );
         _safeMint(_msgSender(), tokenId);
     }
-}
\ No newline at end of file
+
+    function toString(uint256 value) internal pure returns (string memory) {
```

Comment:
Use the one already present in openzeppelin

This is the import for it
import "@openzeppelin/contracts/utils/Strings.sol";